source/unified-test-format/schema-1.4.json

```diff
@@ -55,6 +55,10 @@
             "enum": ["single", "replicaset", "sharded", "sharded-replicaset", "load-balanced"]
           }
         },
+        "serverless": {
+          "type": "string",
+          "enum": ["require", "forbid", "allow"]
```

Comment:
Please add two `invalid/` tests:

 * runOnRequirement-serverless-type.yml with a non-string value
 * runOnRequirement-serverless-enum.yml with an unacceptable string value

There should be enough prior art to base these off of.